### ISSUE TITLE
Do not lose all changes made during debugging

### ISSUE THREAD

**Issue by xusheng6** [Vector35/debugger#213]
Right now, all changes are lost after the debugging ends. This is non-intuitive and counterproductive. We should offer to copy changes made to the debugger view to the original view. Optionally, we can allow the user to apply some of the changes

**Comment by xusheng6** [comment]
Somehow related to https://github.com/Vector35/binaryninja-api/issues/2882

**Comment by xusheng6** [comment]
Related to https://github.com/Vector35/binaryninja-api/issues/920. I am not entirely sure whether it blocks this issue -- I might be able to workaround it since the debugger binary view is already storing the memory content in a different way

**Comment by xusheng6** [comment]
I have updated the issue title. This issue will likely be resolved soon.

We have decided against the originally planned approach. Instead, we are not adding an overlay to the base view, e.g., the PE view to represnet the entire memory space of the running process. And all analysis objects are still attached to the PE view, not the debugger view. So there is no longer a need to copy the changes between binary views at all
